include/NovelRT/NovelRT.h

```diff
@@ -184,9 +178,6 @@
 
   // Utilities
   #include <NovelRT.Interop/Utilities/NrtMisc.h>
-
-  // Windowing
-  #include <NovelRT.Interop/Windowing/NrtWindowingService.h>
```

Comment:
Why is this in the PR?

Reply:
Related to #484, else using the `NovelRT.h` header with Interop would just not compile which is sad.

Reply:
Its already merged in main though?